src/components/elements/off-canvas-menu.js

```diff
@@ -57,7 +58,15 @@ const OffCanvasMenu = styled.section`
     transition: transform 0.4s;
     box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.1);
     left: -254px;
-    ${({ is_canvas_menu_open }) => is_canvas_menu_open && 'transform: translateX(254px)'};
+    ${({ is_canvas_menu_open, is_rtl }) => {
```

Comment:
Can we make a function for this for these translate issues?

Reply:
added `withLangDirection` function, thanks thanks :pray: